### ISSUE TITLE
Represent `.mailmap` file of repo as a queryable table

### ISSUE THREAD

**Issue by chrisma** [mergestat/mergestat#254]
What do you think of the idea of the parsed `.mailmap` file from the repos (if there is one) being represented by a queryable table in MergeStat?

This would allow constructing a query to identify all email addresses of contributors that have not been assigned a canonical name through the `.mailmap` file.

**Comment by patrickdevivo** [comment]
I think that could make sense - so something like `SELECT * FROM mailmap('mailmap-file-contents')` which would return rows of "proper" name/emails and their corresponding values to match on

**Comment by chrisma** [comment]
Yes, `SELECT * FROM mailmap('path/to/repo')` might be even easier to use, if we assume the convention that a `.mailmap` file (if there is one) is included in the repo root.

| canonical_name | email                       |
|----------------|-----------------------------|
| ProperNameA    | A@example.com               |
| ProperNameA    | also-A@example.com          |
| ProperNameA    | another-A-alias@example.com |
| ProperNameB    | B@example.com               |
| ProperNameB    | B@users.noreply.github.com  |
